force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -226,20 +254,26 @@ public with sharing class ClassesObjectsMethods {
      * where you need to manipulate and manage book information.
      */
     public class Book {
+        public String title;
+        public String author;
+        public Integer pages;
         // Declare three public instance variables - title, author, pages
         // title
         // author
         // pages
     
         // Book constructor to initialize the title and author
-        public Book() {
+        public Book(String bookTitle, String bookAuthor) {
+            this.title = bookTitle;
+            this.author = bookAuthor;
             // Initialize the title and author
         }
     
         // Method that returns the details of the book
         public String getBookDetails() {
+            
             // return A string containing the details of the book in the format "Title: <title>, Author: <author>"
-            return null; // Replace null with the variable you used to store the result
+            return 'Title: '+ title + ', Author: ' + author; 
```

Comment:
```suggestion
            return 'Title: '+ this.title + ', Author: ' + this.author; 
```
Although the 'this' keyword is not required in apex, it's good practice to use it.